var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -148,6 +167,11 @@ class Cp2k(MakefilePackage, CMakePackage, CudaPackage, ROCmPackage):
             description="Use CUBLAS for general matrix operations in DBCSR",
         )
 
+    depends_on("hipcc", when="+hip_backend+cuda")
```

Comment:
Would
```suggestion
    depends_on("hipcc", when="+hip_backend")
```
be enough, since `+cuda` is implied by `+hip_backend`?

Reply:
yeah `+hip_backend+cuda` is redundant. I will change this ASAP